dotnet/src/SemanticKernel.Abstractions/Data/VectorSearchFiltering/FilterClauseType.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
```

Comment:
FYI: I dropped this in the `ITextSearch` abstractions, but we can consolidate this stuff later

Reply:
Sounds good.  Probably makes sense to do the same for the SearchQuery types as well then, since I can use the type of the query to do everything.  I'll make a note of it.